spec.bs

```diff
@@ -22,6 +22,16 @@ Assume Explicit For: yes
 <pre class="anchors">
 urlPrefix: https://www.ietf.org/rfc/rfc4122.txt
   type: dfn; text: urn uuid
+spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/

```

Comment:
(link to the single-page spec, you coward 😎 )

Reply:
LOL, I'd prefer to normally but there's two reasons to use multipage here:
 1. To fit in with the other cool kids [NavigationAPI](https://raw.githubusercontent.com/WICG/navigation-api/91c2e7f959418d71e26e5b9a6723fed2944001d9/spec.bs), [Prerendering](https://raw.githubusercontent.com/WICG/nav-speculation/main/prerendering.bs), and [Prefetch](https://raw.githubusercontent.com/WICG/nav-speculation/main/prefetch.bs)
 2. I prefer single-page links when I copy and paste them into my HTML Standard tab so I get instant navigation. But just clicking them from a spec and having to load them in a new tab is a little painful :/

However, I did simplify this a bit by not breaking it down by each individual page link here, and just using the global `/C` affix before each hash.

Reply:
Sorry I'm totally just kidding here. :)